README.md

```diff
@@ -56,6 +56,10 @@ Add the following to your Podfile and run `$ pod install`:
       compile project(':react-native-sensors')
     ```
 
+## Running on Simulator
+
+Currently, both iOS and Android simulators offer no support for sensors. In order to retrieve any sensor output, you **must develop on a real device**.
```

Comment:
this is not entirely true, you can emulate _some_ sensors on android, 
![image](https://user-images.githubusercontent.com/7002833/40780424-f088206e-64e0-11e8-872c-370863ab8189.png)

Reply:
https://stackoverflow.com/a/40160228

Reply:
WHAAAAAAT, that is new to me, super awesome <3 
@barbsicle Could you extend your change?